packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -20,7 +20,30 @@ const BlockMover = ( {
 	onMoveUp,
 	firstIndex,
 	rootClientId,
+	horizontalDirection,
 } ) => {
+	const firstButtonIcon = horizontalDirection ? arrowLeft : arrowUp;
```

Comment:
Please check this comment - https://github.com/WordPress/gutenberg/pull/19013#discussion_r401783619